### PR TITLE
build on darwin :^) (at least without `--sysroot`)

### PR DESCRIPTION
**Pull request by xvanc**

### CODE REVIEW

**Comment by Andy-Python-Programmer** [comment]
based

File: aero.py [hunk 1]
```diff
@@ -445,23 +455,54 @@ def run_in_emulator(args, iso_path):
...
+    Shell out to sysctl(1)
+
+    Returns the value as a string.
+    Non-leaf nodes will return the value for each sub-node separated by newline characters.
+    """
+    status, stdout, stderr = run_command(["sysctl", "-n", name],
+                                         stdout=subprocess.PIPE,
+                                         stderr=subprocess.PIPE)
+    if status != 0:
+        print("`sysctl` failed:")
```
Comment (Andy-Python-Programmer) [inline aero.py:480]: Maybe make this:

```python
print("`sysctl` failed: ", end="")
print(stderr.decode())
```

This way the full error can be on the same line ;^)

Comment (xvanc) [reply aero.py:480]: on it

**Comment by Andy-Python-Programmer** [comment]
Thanks @xvanc!

**Pull request merged**